src/main/java/org/commcare/formplayer/database/models/FormplayerCaseSearchIndexTable.kt

```diff
@@ -0,0 +1,12 @@
+package org.commcare.formplayer.database.models
+
+import org.commcare.formplayer.services.ConnectionHandler
+
+class FormplayerCaseSearchIndexTable(connectionHandler: ConnectionHandler, private val searchKey: String) :
+    FormplayerCaseIndexTable(connectionHandler, false) {
+
+    private val CASE_SEARCH_INDEX_STORAGE_TABLE_PREFIX = "case_search_index_storage_FOR_"
+    override fun getTableName(): String {
+        return CASE_SEARCH_INDEX_STORAGE_TABLE_PREFIX + searchKey
```

Comment:
this could also be a class property that's set in the constructor

Reply:
https://github.com/dimagi/formplayer/pull/1351/commits/ffac6272990c50aa396c264ddcfd6eae5e9192a5